### ISSUE TITLE
CREATE TABLE should check for duplicate columns

### ISSUE THREAD

**Issue by electrum** [prestodb/presto#7846]
`CreateTableTask` should check for a duplicate column and throw `SemanticException(DUPLICATE_COLUMN_NAME)` rather than leaving it up to the connector (which should not be expected to handle this).

Here is what happens today against the Hive connector:

```
presto> create table test (x bigint, x bigint);
Query 20170417_183416_15485_qs2vr failed: Multiple entries with same key: x=HiveColumnHandle{clientId=prism, name=x, hiveType=bigint, hiveColumnIndex=1, columnType=REGULAR, comment=Optional.empty} and x=HiveColumnHandle{clientId=prism, name=x, hiveType=bigint, hiveColumnIndex=0, columnType=REGULAR, comment=Optional.empty}. To index multiple values under a key, use Multimaps.index.
```

This should have the same error message as `CREATE TABLE AS`:

```
presto:di> create table test as select 123 x, 456 x;
Query 20170417_183658_15553_qs2vr failed: line 1:1: Column name 'x' specified more than once
```

**Comment by ebyhr** [comment]
How about CREATE/REPLACE VIEW?

(e.g.) The query fails in Teradata.
```
create table t1 (id integer);
replace view v2 (id, id) as select id, id from t1;
```

```
REPLACE VIEW Failed. 3515:  Duplication of column ID in a table, derived table, view, macro or trigger. 
```

**Comment by ebyhr** [comment]
Sorry, I found the test already exists in `TestAnalyzer#testCreateTableAsColumns`.

**Comment by medo** [comment]
@electrum I would like to create a pull request for this task

**Comment by ebyhr** [comment]
Hi @medo, I already sent PR 3 days ago. #7869

**Comment by medo** [comment]
@ebyhr thanks, I will find another issue then :)

**Comment by haozhun** [comment]
Fixed in #7869
